app/consapp/rtkrcv/rtkrcv.c

```diff
@@ -178,9 +178,9 @@ static const char *pathopts[]={         /* path options help */
 #define TIMOPT  "0:gpst,1:utc,2:jst,3:tow"
 #define CONOPT  "0:dms,1:deg,2:xyz,3:enu,4:pyl"
 #define FLGOPT  "0:off,1:std+2:age/ratio/ns"
-#define ISTOPT  "0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http"
+#define ISTOPT  "0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,6:ntripcli,7:ftp,8:http"
 #define OSTOPT  "0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,6:ntripsvr,11:ntripc_c"
-#define FMTOPT  "0:rtcm2,1:rtcm3,2:oem4,3:oem3,4:ubx,5:ss2,6:hemis,7:skytraq,8:gw10,9:javad,10:nvs,11:binex,12:rt17,13:sbf,14:cmr,15:tersus,18:sp3"
+#define FMTOPT  "0:rtcm2,1:rtcm3,2:oem4,3:oem3,4:ubx,5:ss2,6:hemis,7:skytraq,8:javad,9:nvs,10:binex,12:rt17,13:sbf,14:cmr,15:tersus,18:sp3"
```

Comment:
is there no 11?

Reply:
the current support for GPS, GAL and BDS L1/L5 processing doesn't (i think) require to consider the other formats other than `rtcm` and `binex`. In our master in https://github.com/swift-nav/RTKLIB/blob/96fe1214bb70c017ccd73e23fc5c24f433ff6a3e/app/rtkrcv/rtkrcv.c#L183, the 5 and 6 format options don't agree with the format definition in https://github.com/swift-nav/RTKLIB/blob/96fe1214bb70c017ccd73e23fc5c24f433ff6a3e/src/rtklib.h#L437-L438, so i think we will need to do extra work to properly make this fully working.

Reply:
thats unfortunate we have drifted from master, maybe we should make a ticket to look at syncing this to make future rebasing easier

Reply:
yeah, this ticket already considers that, https://swift-nav.atlassian.net/browse/DEVINFRA-560.